src/Symfony/Component/Runtime/Runner/Psr/LaminasEmitter.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Runtime\Runner\Psr;
+
+use Laminas\HttpHandlerRunner\Emitter\SapiEmitter;
+use Psr\Http\Message\ResponseInterface;
+use Psr\Http\Message\ServerRequestInterface;
+use Psr\Http\Server\RequestHandlerInterface;
+use Symfony\Component\Runtime\RunnerInterface;
+
+/**
+ * @author Tobias Nyholm <tobias.nyholm@gmail.com>
+ *
+ * @experimental in 5.3
+ */
+class LaminasEmitter implements RunnerInterface
+{
+    private $requestHandler;
+    private $response;
+    private $request;
+    private $emitter;
+
+    private function __construct()
+    {
+        if (!class_exists(SapiEmitter::class)) {
+            throw new \LogicException(sprintf('The "%s" class requires "laminas/laminas-httphandlerrunner". Try running "composer require laminas/laminas-httphandlerrunner".', self::class));
+        }
+
+        $this->emitter = new SapiEmitter();
```

Comment:
I have a proxy that conditionally needs to use `SapiStreamEmitter` instead of `SapiEmitter`. It would be great to have support for this option.

See: https://docs.laminas.dev/laminas-httphandlerrunner/emitters/#conditionally-using-the-sapistreamemitter

Reply:
Great idea. I've added the `laminas_emitter` option.